### DISCUSSION TITLE
What is this?

### DISCUSSION THREAD

**Discussion by iao14** [Darkshadogt/Dark-Auto#1] [Show and tell]
I don't know what this supposed to be, but i don't feel safe if i run this program...

![image](https://github.com/Darkshadogt/Dark-Auto/assets/129288541/0e578f71-6e19-4360-9314-2bb095cdfdf7)

**Comment by Darkshadogt** [comment]
I used auto-py-to-exe and Pyinstaller to convert the py file to exe file. This issue is well known with Pyinstaller which many anti-virus programs detect the exe file as false positives.
